content/docs/authorization/_index.md

```diff
@@ -33,7 +33,7 @@ The following diagram shows a high-level overview of CSM for Authorization with
 {{<table "table table-striped table-bordered table-sm">}}
 | COP/OS | Supported Versions |
 |-|-|
-| Kubernetes    | 1.25, 1.26, 1.27 |
+| Kubernetes    | 1.25, 1.26, 1.27, 1.28 |
 | RHEL          |     7.x, 8.x     |
```

Comment:
We no longer specify OS support so RHEL and CentOS rows can be removed

Reply:
> We no longer specify OS support so RHEL and CentOS rows can be removed

Fixed